bench/projects/Bench.AutofacApiServer/appsettings.Development.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
Why override the defaults with... the defaults?

Reply:
Blame the default template for that one...